### PR TITLE
Update CreateExp and UpdateResults docs to include JSON keys details

### LINKED ISSUE #657: Update docs explaining the json keys for createExperiment & updateResults
Update docs explaining the json keys for createExperiment & updateResults.  Also update mandatory field details

- https://github.com/kruize/autotune/blob/master/design/CreateExperiment.md
- https://github.com/kruize/autotune/blob/master/design/UpdateResults.md

### PR DESCRIPTION
**Pull request by khansaad**
This PR fixes #657

### CODE REVIEW

**Comment by khansaad** [comment]
@chandrams @dinogun I have added the details of the keys. Need confirmation on the mandatory keys.

File: design/CreateExperiment.md [hunk 1]
```diff
@@ -5,12 +5,40 @@ tuned.
...
+    - **container_image_name** \
+      Name of the image deployed in the container. E.g: `kruize/tfb-db:0.1`
+- **trial_settings** \
+    contains details of the trials like the `measurement_duration`
+  - **measurement_duration** \
+    Value in mins depicting the duration of monitoring. E.g: `15min`  
+- **recommendation_settings** \
+  contains details of the recommendations like the `threshold value`
+  - **threshold** \
+    Double value depicting the threshold value of the recommendation. E.g: `0.1`
```
[the remark this answers is not in the document]
Comment (khansaad) [reply design/CreateExperiment.md:42]: Updated

File: design/CreateExperiment.md [hunk 2]
```diff
@@ -5,12 +5,40 @@ tuned.
...
+- **performance_profile** \
+  Performance profile name which contains the `SLO` data and the `objective function` data. 
+- **mode** \
+  It can be either `monitor` or `experiment` 
+- **target_cluster** \
+   Target cluster can be one of `remote` or `local` 
+- **kubernetes_objects** \
+   contains details of the deployment, containers and namespace of the application to be monitored. 
+  - **type** \
+    A type can either be a `Deployment`, `Replicaset`, `Statefulset`, etc.
```
[the remark this answers is not in the document]
Comment (khansaad) [reply design/CreateExperiment.md]: Updated

File: design/CreateExperiment.md [hunk 3]
```diff
@@ -5,29 +5,73 @@ tuned.
...
-* performance_profile \
-  `performance_profile` is a pre-defined performance objective that is baked into Kruize.
+- **cluster_name** \
+  Name of the cluster where the app is deployed.
 
+### Performance Objective
+- **performance_profile** \
+    `performance_profile` is a pre-defined performance objective that is baked into Kruize. 
+- **slo** \
+  Service Level Objective to be defined in the `performance_profile` as shown in [Performance Profile](https://github.com/kruize/autotune/blob/master/manifests/autotune/performance-profiles/resource_optimization_openshift.yaml) section, or it can be defined here in the experiment JSON and a Default Performance Profile will be created.
```
Comment (dinogun) [inline design/CreateExperiment.md:20]: This is not the correct description. Please add more details here to explain SLO better. See the [README.md](https://github.com/kruize/kruize-demos/blob/main/README.md) for reference.

File: design/CreateExperiment.md [hunk 4]
```diff
@@ -5,29 +5,73 @@ tuned.
...
+- **mode** \
+  It can be either `monitor` or `experiment` 
+- **target_cluster** \
+   Target cluster can be one of `remote` or `local`
+
+### Experiment Target
+- **kubernetes_objects** \
+   contains details of the deployment, containers and namespace of the application to be monitored. 
+  - **type** \
+    A type can either be a `Deployment`, `Replicaset`, `Statefulset`, `Deployment_Config`, `Replication_Controller`,
```
Comment (dinogun) [inline design/CreateExperiment.md:48]: There are no underscores, please add the correct names

File: design/CreateExperiment.md [hunk 5]
```diff
@@ -5,29 +5,73 @@ tuned.
...
+   Target cluster can be one of `remote` or `local`
+
+### Experiment Target
+- **kubernetes_objects** \
+   contains details of the deployment, containers and namespace of the application to be monitored. 
+  - **type** \
+    A type can either be a `Deployment`, `Replicaset`, `Statefulset`, `Deployment_Config`, `Replication_Controller`, 
+    `Daemonset`, etc.
+  - **name** \
+    A string name is specified for identifying individual deployments. Each deployment can have multiple experiments.
```
Comment (dinogun) [inline design/CreateExperiment.md:51]: Not sure what you mean by `Each deployment can have multiple experiments`? We currently support only one experiment. Also this should not be called `deployment`

File: design/CreateExperiment.md (hunk 3 above)
Comment (khansaad) [reply design/CreateExperiment.md]: Added a detailed description

**Comment by dinogun** [comment]
Closing stale PRs

**Pull request closed without merging**